project-keeper/src/main/resources/templates/.github/workflows/ci-build.yml

```diff
@@ -110,6 +110,78 @@ jobs:
           path: ${{ steps.build-pk-verify.outputs.release-artifacts }}
           retention-days: 5
 
+      - name: Configure link check
+        id: configure-link-check
+        run: |
+          mkdir -p ./target
+          echo '{"aliveStatusCodes": [429, 200], "ignorePatterns": [' \
+               '{"pattern": "^https?://(www|dev).mysql.com/"},' \
+               '{"pattern": "^https?://(www.)?opensource.org"}' \
+               '{"pattern": "^https?://(www.)?eclipse.org"}' \
+               '{"pattern": "^https?://projects.eclipse.org"}' \
+               ']}' > ./target/broken_links_checker.json
+      - uses: gaurav-nelson/github-action-markdown-link-check@v1
+        id: run-link-check
+        with:
+          use-quiet-mode: "yes"
+          use-verbose-mode: "yes"
+          config-file: ./target/broken_links_checker.json
+
+  next-java-compatibility:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: "bash"
+    permissions:
+      contents: read
+    concurrency:
+      group: ${{ github.workflow }}-next-java-${{ github.ref }}
+      cancel-in-progress: true
+    steps:
+      - name: Checkout the repository
+        id: checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: Set up JDK $nextJavaVersion
+        id: setup-java
+        uses: actions/setup-java@v4
+        with:
+          distribution: "temurin"
+          java-version: $nextJavaVersion
+          cache: "maven"
+      - name: Run tests and build with Maven $nextJavaVersion
+        id: build-next-java
+        run: mvn --batch-mode clean package -DtrimStackTrace=false -Djava.version=$nextJavaVersion
+
+  build:
+    needs:
+      - build-and-test
+      - next-java-compatibility
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        shell: "bash"
+    permissions:
+      contents: read
+      issues: read # Required for PK verify-release
+    outputs:
+      release-required: ${{ steps.check-release.outputs.release-required }}
+    steps:
+      - name: Checkout the repository
+        id: checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: Set up JDKs
+        id: setup-java
+        uses: actions/setup-java@v4
+        with:
+          distribution: "temurin"
+          java-version: |
+            11
+            17
```

Comment:
Shouldn't these versions be defined by the entries in `.project-keeper.yml`?

Reply:
These versions are overwritten by PK based on property `java.version` in the `pom.xml`.